### DISCUSSION TITLE
[DONE ✅]  Make PayJoin default for hot wallet

### DISCUSSION THREAD

**Discussion by ghost** [btcpayserver/btcpayserver#2406] [Ideas / Feature Requests]
If the user opts-in to run a hot wallet, then the PayJoin support should be activated by default. He already agreed to the risk of a hot wallet, so defaulting to PayJoin should not be an issue.

[cc @Kukks]

**Comment by dennisreimann** [comment]
Not sure if we should make it the default, but offering the option to opt-in during the creation would be a good way to promote this feature.

**Reply by ghost** [reply]
well already now there's the opt-in to use PayJoin with any hot wallet.
but the defaults matter, and especially for the ambiguity point of PayJoin, a larger receiver user base would be very valuable.

**Reply by dstrukt** [reply]
Would be good to get consensus on this as a default, but I with out a doubt support adding it as an opt-in option.

**Reply by Kixunil** [reply]
How about a check box during wallet creation "Enable PayJoin" checked by default?

**Reply by dstrukt** [reply]
+1 

That's probably the most sensible way to handle this.

We should also probably have a (?) or short bit of context to help provide info on something that will be enabled by default.

**Reply by Kixunil** [reply]
I agree with (?) icon

**Comment by Zaxounette** [comment]
FYI, this was brought up in [Wasabi Research Club #23](https://www.youtube.com/watch?v=oTi0z1Sghp8). ~22 minutes from start.

**Reply by ghost** [reply]
The reason why free software wins?
Contributors open feature requests while doing a live stream about the feature. 
And other contributors of the project actually listen to the show and post the link live as well.

Lovely :black_heart: :smile:

**Comment by Kukks** [comment]
ACK from my end, the end-user loses nothing out of this.

**Comment by dennisreimann** [comment]
I can prepare this based on what's been discussed here, so that we can narrow it down in the next design meeting :)

**Reply by dennisreimann** [reply]
What about this? If we enable it by default I'd recommend to show it immediately and not hide it in the advanced settings, though it feels like it would naturaly belong there.

![hotwallet-payjoin](https://user-images.githubusercontent.com/886/114045630-702e7880-9888-11eb-998f-562147f27ba0.png)

Wording for the accompanying text tbd, the help icon links to the [PayJoin docs](https://docs.btcpayserver.org/Payjoin/).

**Reply by Zaxounette** [reply]
Add a "recommended" on the right of the enable button and for me, it's good to go, including your accompanying text.

**Reply by dennisreimann** [reply]
Added a draft PR. Let's continue the discussion in #2450.
